test/extended/cli/explain.go

```diff
@@ -372,11 +373,13 @@ var _ = g.Describe("[sig-cli] oc explain networking types", func() {
 	oc := exutil.NewCLI("oc-explain", exutil.KubeConfigPath())
 
 	g.It("should contain proper fields description for special networking types", func() {
-		for _, st := range specialNetworkingTypes {
-			e2e.Logf("Checking %s, Field=%s...", st.gv, st.field)
-			o.Expect(verifyExplain(oc, nil, schema.GroupVersionResource{},
-				st.pattern, st.field, fmt.Sprintf("--api-version=%s", st.gv))).NotTo(o.HaveOccurred())
-		}
+		networking.InOpenShiftSDNContext(func() {
+			for _, st := range specialNetworkingTypes {
+				e2e.Logf("Checking %s, Field=%s...", st.gv, st.field)
+				o.Expect(verifyExplain(oc, nil, schema.GroupVersionResource{},
+					st.pattern, st.field, fmt.Sprintf("--api-version=%s", st.gv))).NotTo(o.HaveOccurred())
+			}
+		})
```

Comment:
Please add the correct explain rules for OVN types.  All product CRDs are required to have schemas.

Reply:
Each type has it's own idiosyncrasies. They don't have a Spec and Status as a rule. Whatever we write will be very specific.

Reply:
ovn-kubernetes does not create any CRDs. (It stores its private information in the OVN databases rather than in kubernetes.) If we do create CRDs for it in the future we'll create them "correctly" (spec and status, etc) so they don't need to be tested as exceptions.

Reply:
@smarterclayton ^^^

Reply:
That's all I meant.  While we're in here fixing it, run the test for each type and say "doesn't need explain" or "here are the exceptions and the bugs to fix".  If a particular network doesn't have CRDs, say so in code.

Reply:
`network.operator.openshift.io` was the only networking-related type listed-but-commented-out in explain.go, and I've tested it and uncommented it.

So, you mean you want every CRD type that the CNO ever creates, directly or indirectly, to be added to this list, not just CRDs for types in `openshift/api`? I don't have time to do that now but I can create a jira card about it...

Reply:
Yeah, that really sounds like a completely separate PR as it won't solve the current problem of unbreaking the CI tests that the switch to SIG-prefixed tests broke.

Reply:
Yes, you can fix that separately.

CRDS must have schemas.  This is a product requirement.  We can't give customers an empty JSON object and tell them good luck.  It definitely needs a card, and it needs priority, and if you need help, just tell me and I'll get some folks reassigned from other teams to help.

Reply:
filed https://issues.redhat.com/browse/SDN-873